apiserver/server.go

```diff
@@ -43,6 +43,7 @@ func New(cfg Config) *echo.Echo {
 	e.GET("/forecast_events", ForecastEventsHandler(cfg.Store))
 	e.GET("/usage_events", UsageEventsHandler(cfg.Store, cfg.Authenticator))
 	e.GET("/billable_events", BillableEventsHandler(cfg.Store, cfg.Store, cfg.Authenticator))
+	e.GET("/costs", TotalCostHandler(cfg.Store))
```

Comment:
I wonder if `/totals`, `/total_costs`, or `/total_costs_by_plan` would be a better name for the endpoint?